### PR TITLE
Add GitHub trigger and GitHub list workflow commands

### PR DESCRIPTION
**Pull request by MosheEichler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/CIAC-5322

## Description
Added 2 commands
1. command to trigger a workflow (nightly) on GitHub.
2. command to retrieve a list of workflows from GitHub.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/GitHub/Integrations/GitHub/GitHub.yml [hunk 1]
```diff
@@ -4034,7 +4034,111 @@ script:
...
+      description: The commit sha on which the workflow ran.
+      type: String
+    - contextPath: GitHub.Workflow.path
+      description: The GitHub workflow name path
+      type: String
+    - contextPath: GitHub.Workflow.display_title
+      description: The GitHub workflow title
+      type: String
+    - contextPath: GitHub.Workflow.run_number
+      description: The GitHub workflow run number
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/GitHub.yml:4115]:
```suggestion
      description: The GitHub workflow run number.
```

File: Packs/GitHub/Integrations/GitHub/GitHub.yml [hunk 2]
```diff
@@ -4034,7 +4034,111 @@ script:
...
+      type: Number
+    - contextPath: GitHub.Workflow.event
+      description: The GitHub workflow trigger type (scheduled, dispatch)
+      type: String
+    - contextPath: GitHub.Workflow.status
+      description: The GitHub workflow status (in_progress, completed)
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/GitHub.yml:4121]:
```suggestion
      description: The GitHub workflow status (in_progress, completed).
```

File: Packs/GitHub/Integrations/GitHub/GitHub.yml [hunk 3]
```diff
@@ -4034,7 +4034,111 @@ script:
...
+      type: String
+    - contextPath: GitHub.Workflow.conclusion
+      description: The GitHub workflow conclusion (cancelled, success)
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/GitHub.yml:4124]:
```suggestion
      description: The GitHub workflow conclusion (cancelled, success).
```

File: Packs/GitHub/Integrations/GitHub/GitHub.yml [hunk 4]
```diff
@@ -4034,7 +4034,111 @@ script:
...
+      type: String
+    - contextPath: GitHub.Workflow.workflow_id
+      description: The GitHub workflow id (per workflow)
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/GitHub.yml:4127]:
```suggestion
      description: The GitHub workflow ID (per workflow).
```

File: Packs/GitHub/Integrations/GitHub/GitHub.yml [hunk 5]
```diff
@@ -4034,7 +4034,111 @@ script:
...
+      type: String
+    - contextPath: GitHub.Workflow.url
+      description: The GitHub workflow API URL
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/GitHub.yml:4130]:
```suggestion
      description: The GitHub workflow API URL.
```

File: Packs/GitHub/Integrations/GitHub/GitHub.yml [hunk 6]
```diff
@@ -4034,7 +4034,111 @@ script:
...
+      type: String
+    - contextPath: GitHub.Workflow.html_url
+      description: The GitHub workflow html URL
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/GitHub.yml:4133]:
```suggestion
      description: The GitHub workflow HTML URL.
```

File: Packs/GitHub/Integrations/GitHub/GitHub.yml [hunk 7]
```diff
@@ -4034,7 +4034,111 @@ script:
...
+      type: String
+    - contextPath: GitHub.Workflow.created_at
+      description: Datetime the GitHub workflow was created at
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/GitHub.yml:4136]:
```suggestion
      description: Datetime the GitHub workflow was created at.
```

File: Packs/GitHub/Integrations/GitHub/GitHub.yml [hunk 8]
```diff
@@ -4034,7 +4034,111 @@ script:
...
+      type: Date
+    - contextPath: GitHub.Workflow.updated_at
+      description: Datetime the GitHub workflow was updated at
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/GitHub.yml:4139]:
```suggestion
      description: Datetime the GitHub workflow was updated at.
```

File: Packs/GitHub/Integrations/GitHub/README.md [hunk 9]
```diff
@@ -2889,3 +2889,96 @@ Adds up to 10 assignees to an issue. Users already assigned to an issue are not
...
+| owner | The GitHub owner (organization or username) of the repository. | Optional | 
+| repository | The GitHub repository name. | Optional | 
+| workflow | The name of your workflow file. | Required | 
+| limit | The number of workflows to return. Default is 100. | Optional | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| GitHub.Workflow.id | Number | The GitHub workflow id \(per run\) |
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/README.md:2935]:
```suggestion
| GitHub.Workflow.id | Number | The GitHub workflow id \(per run\). | 
```

File: Packs/GitHub/Integrations/GitHub/README.md [hunk 10]
```diff
@@ -2889,3 +2889,96 @@ Adds up to 10 assignees to an issue. Users already assigned to an issue are not
...
+| repository | The GitHub repository name. | Optional | 
+| workflow | The name of your workflow file. | Required | 
+| limit | The number of workflows to return. Default is 100. | Optional | 
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| GitHub.Workflow.id | Number | The GitHub workflow id \(per run\) | 
+| GitHub.Workflow.name | String | The GitHub workflow name |
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/README.md:2936]:
```suggestion
| GitHub.Workflow.name | String | The GitHub workflow name. | 
```

File: Packs/GitHub/Integrations/GitHub/README.md [hunk 11]
```diff
@@ -2889,3 +2889,96 @@ Adds up to 10 assignees to an issue. Users already assigned to an issue are not
...
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| GitHub.Workflow.id | Number | The GitHub workflow id \(per run\) | 
+| GitHub.Workflow.name | String | The GitHub workflow name | 
+| GitHub.Workflow.head_branch | String | The branch on which the workflow ran. | 
+| GitHub.Workflow.head_sha | String | The commit sha on which the workflow ran. | 
+| GitHub.Workflow.path | String | The GitHub workflow name path |
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/README.md:2939]:
```suggestion
| GitHub.Workflow.path | String | The GitHub workflow name path. | 
```

File: Packs/GitHub/Integrations/GitHub/README.md [hunk 12]
```diff
@@ -2889,3 +2889,96 @@ Adds up to 10 assignees to an issue. Users already assigned to an issue are not
...
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| GitHub.Workflow.id | Number | The GitHub workflow id \(per run\) | 
+| GitHub.Workflow.name | String | The GitHub workflow name | 
+| GitHub.Workflow.head_branch | String | The branch on which the workflow ran. | 
+| GitHub.Workflow.head_sha | String | The commit sha on which the workflow ran. | 
+| GitHub.Workflow.path | String | The GitHub workflow name path | 
+| GitHub.Workflow.display_title | String | The GitHub workflow title |
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/README.md:2940]:
```suggestion
| GitHub.Workflow.display_title | String | The GitHub workflow title. | 
```

File: Packs/GitHub/Integrations/GitHub/README.md [hunk 13]
```diff
@@ -2889,3 +2889,96 @@ Adds up to 10 assignees to an issue. Users already assigned to an issue are not
...
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| GitHub.Workflow.id | Number | The GitHub workflow id \(per run\) | 
+| GitHub.Workflow.name | String | The GitHub workflow name | 
+| GitHub.Workflow.head_branch | String | The branch on which the workflow ran. | 
+| GitHub.Workflow.head_sha | String | The commit sha on which the workflow ran. | 
+| GitHub.Workflow.path | String | The GitHub workflow name path | 
+| GitHub.Workflow.display_title | String | The GitHub workflow title | 
+| GitHub.Workflow.run_number | Number | The GitHub workflow run number |
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/README.md:2941]:
```suggestion
| GitHub.Workflow.run_number | Number | The GitHub workflow run number. | 
```

File: Packs/GitHub/Integrations/GitHub/README.md [hunk 14]
```diff
@@ -2889,3 +2889,96 @@ Adds up to 10 assignees to an issue. Users already assigned to an issue are not
...
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| GitHub.Workflow.id | Number | The GitHub workflow id \(per run\) | 
+| GitHub.Workflow.name | String | The GitHub workflow name | 
+| GitHub.Workflow.head_branch | String | The branch on which the workflow ran. | 
+| GitHub.Workflow.head_sha | String | The commit sha on which the workflow ran. | 
+| GitHub.Workflow.path | String | The GitHub workflow name path | 
+| GitHub.Workflow.display_title | String | The GitHub workflow title | 
+| GitHub.Workflow.run_number | Number | The GitHub workflow run number | 
+| GitHub.Workflow.event | String | The GitHub workflow trigger type \(scheduled, dispatch\) |
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/README.md:2942]:
```suggestion
| GitHub.Workflow.event | String | The GitHub workflow trigger type \(scheduled, dispatch\). | 
```

File: Packs/GitHub/Integrations/GitHub/README.md [hunk 15]
```diff
@@ -2889,3 +2889,96 @@ Adds up to 10 assignees to an issue. Users already assigned to an issue are not
...
+| --- | --- | --- |
+| GitHub.Workflow.id | Number | The GitHub workflow id \(per run\) | 
+| GitHub.Workflow.name | String | The GitHub workflow name | 
+| GitHub.Workflow.head_branch | String | The branch on which the workflow ran. | 
+| GitHub.Workflow.head_sha | String | The commit sha on which the workflow ran. | 
+| GitHub.Workflow.path | String | The GitHub workflow name path | 
+| GitHub.Workflow.display_title | String | The GitHub workflow title | 
+| GitHub.Workflow.run_number | Number | The GitHub workflow run number | 
+| GitHub.Workflow.event | String | The GitHub workflow trigger type \(scheduled, dispatch\) | 
+| GitHub.Workflow.status | String | The GitHub workflow status \(in_progress, completed\) |
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/README.md:2943]:
```suggestion
| GitHub.Workflow.status | String | The GitHub workflow status \(in_progress, completed\). | 
```

File: Packs/GitHub/Integrations/GitHub/README.md [hunk 16]
```diff
@@ -2889,3 +2889,96 @@ Adds up to 10 assignees to an issue. Users already assigned to an issue are not
...
+| GitHub.Workflow.id | Number | The GitHub workflow id \(per run\) | 
+| GitHub.Workflow.name | String | The GitHub workflow name | 
+| GitHub.Workflow.head_branch | String | The branch on which the workflow ran. | 
+| GitHub.Workflow.head_sha | String | The commit sha on which the workflow ran. | 
+| GitHub.Workflow.path | String | The GitHub workflow name path | 
+| GitHub.Workflow.display_title | String | The GitHub workflow title | 
+| GitHub.Workflow.run_number | Number | The GitHub workflow run number | 
+| GitHub.Workflow.event | String | The GitHub workflow trigger type \(scheduled, dispatch\) | 
+| GitHub.Workflow.status | String | The GitHub workflow status \(in_progress, completed\) | 
+| GitHub.Workflow.conclusion | String | The GitHub workflow conclusion \(cancelled, success\) |
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/README.md:2944]:
```suggestion
| GitHub.Workflow.conclusion | String | The GitHub workflow conclusion \(cancelled, success\). | 
```

File: Packs/GitHub/Integrations/GitHub/README.md [hunk 17]
```diff
@@ -2889,3 +2889,96 @@ Adds up to 10 assignees to an issue. Users already assigned to an issue are not
...
+| GitHub.Workflow.name | String | The GitHub workflow name | 
+| GitHub.Workflow.head_branch | String | The branch on which the workflow ran. | 
+| GitHub.Workflow.head_sha | String | The commit sha on which the workflow ran. | 
+| GitHub.Workflow.path | String | The GitHub workflow name path | 
+| GitHub.Workflow.display_title | String | The GitHub workflow title | 
+| GitHub.Workflow.run_number | Number | The GitHub workflow run number | 
+| GitHub.Workflow.event | String | The GitHub workflow trigger type \(scheduled, dispatch\) | 
+| GitHub.Workflow.status | String | The GitHub workflow status \(in_progress, completed\) | 
+| GitHub.Workflow.conclusion | String | The GitHub workflow conclusion \(cancelled, success\) | 
+| GitHub.Workflow.workflow_id | String | The GitHub workflow id \(per workflow\) |
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/README.md:2945]:
```suggestion
| GitHub.Workflow.workflow_id | String | The GitHub workflow ID \(per workflow\). | 
```

File: Packs/GitHub/Integrations/GitHub/README.md [hunk 18]
```diff
@@ -2889,3 +2889,96 @@ Adds up to 10 assignees to an issue. Users already assigned to an issue are not
...
+| GitHub.Workflow.head_branch | String | The branch on which the workflow ran. | 
+| GitHub.Workflow.head_sha | String | The commit sha on which the workflow ran. | 
+| GitHub.Workflow.path | String | The GitHub workflow name path | 
+| GitHub.Workflow.display_title | String | The GitHub workflow title | 
+| GitHub.Workflow.run_number | Number | The GitHub workflow run number | 
+| GitHub.Workflow.event | String | The GitHub workflow trigger type \(scheduled, dispatch\) | 
+| GitHub.Workflow.status | String | The GitHub workflow status \(in_progress, completed\) | 
+| GitHub.Workflow.conclusion | String | The GitHub workflow conclusion \(cancelled, success\) | 
+| GitHub.Workflow.workflow_id | String | The GitHub workflow id \(per workflow\) | 
+| GitHub.Workflow.url | String | The GitHub workflow API URL |
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/README.md:2946]:
```suggestion
| GitHub.Workflow.url | String | The GitHub workflow API URL. | 
```

File: Packs/GitHub/Integrations/GitHub/README.md [hunk 19]
```diff
@@ -2889,3 +2889,96 @@ Adds up to 10 assignees to an issue. Users already assigned to an issue are not
...
+| GitHub.Workflow.head_sha | String | The commit sha on which the workflow ran. | 
+| GitHub.Workflow.path | String | The GitHub workflow name path | 
+| GitHub.Workflow.display_title | String | The GitHub workflow title | 
+| GitHub.Workflow.run_number | Number | The GitHub workflow run number | 
+| GitHub.Workflow.event | String | The GitHub workflow trigger type \(scheduled, dispatch\) | 
+| GitHub.Workflow.status | String | The GitHub workflow status \(in_progress, completed\) | 
+| GitHub.Workflow.conclusion | String | The GitHub workflow conclusion \(cancelled, success\) | 
+| GitHub.Workflow.workflow_id | String | The GitHub workflow id \(per workflow\) | 
+| GitHub.Workflow.url | String | The GitHub workflow API URL | 
+| GitHub.Workflow.html_url | String | The GitHub workflow html URL |
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/README.md:2947]:
```suggestion
| GitHub.Workflow.html_url | String | The GitHub workflow HTML URL. | 
```

File: Packs/GitHub/Integrations/GitHub/README.md [hunk 20]
```diff
@@ -2889,3 +2889,96 @@ Adds up to 10 assignees to an issue. Users already assigned to an issue are not
...
+| GitHub.Workflow.path | String | The GitHub workflow name path | 
+| GitHub.Workflow.display_title | String | The GitHub workflow title | 
+| GitHub.Workflow.run_number | Number | The GitHub workflow run number | 
+| GitHub.Workflow.event | String | The GitHub workflow trigger type \(scheduled, dispatch\) | 
+| GitHub.Workflow.status | String | The GitHub workflow status \(in_progress, completed\) | 
+| GitHub.Workflow.conclusion | String | The GitHub workflow conclusion \(cancelled, success\) | 
+| GitHub.Workflow.workflow_id | String | The GitHub workflow id \(per workflow\) | 
+| GitHub.Workflow.url | String | The GitHub workflow API URL | 
+| GitHub.Workflow.html_url | String | The GitHub workflow html URL | 
+| GitHub.Workflow.created_at | Date | Datetime the GitHub workflow was created at |
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/README.md:2948]:
```suggestion
| GitHub.Workflow.created_at | Date | Datetime the GitHub workflow was created at. | 
```

File: Packs/GitHub/Integrations/GitHub/README.md [hunk 21]
```diff
@@ -2889,3 +2889,96 @@ Adds up to 10 assignees to an issue. Users already assigned to an issue are not
...
+| GitHub.Workflow.display_title | String | The GitHub workflow title | 
+| GitHub.Workflow.run_number | Number | The GitHub workflow run number | 
+| GitHub.Workflow.event | String | The GitHub workflow trigger type \(scheduled, dispatch\) | 
+| GitHub.Workflow.status | String | The GitHub workflow status \(in_progress, completed\) | 
+| GitHub.Workflow.conclusion | String | The GitHub workflow conclusion \(cancelled, success\) | 
+| GitHub.Workflow.workflow_id | String | The GitHub workflow id \(per workflow\) | 
+| GitHub.Workflow.url | String | The GitHub workflow API URL | 
+| GitHub.Workflow.html_url | String | The GitHub workflow html URL | 
+| GitHub.Workflow.created_at | Date | Datetime the GitHub workflow was created at | 
+| GitHub.Workflow.updated_at | Date | Datetime the GitHub workflow was updated at |
```
Comment (ShirleyDenkberg) [inline Packs/GitHub/Integrations/GitHub/README.md:2949]:
```suggestion
| GitHub.Workflow.updated_at | Date | Datetime the GitHub workflow was updated at. | 
```

**Comment by ShirleyDenkberg** [comment]
@DeanArbel @bziser Doc review completed.

**Review by bziser — APPROVED** [review]
great!

**Pull request merged**